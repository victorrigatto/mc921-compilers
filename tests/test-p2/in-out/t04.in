int a = 100 + 50;
int b = (13 + a) * 10;
