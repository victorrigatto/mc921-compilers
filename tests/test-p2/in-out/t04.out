Program:
    GlobalDecl:
        Decl: ID(name=a)
            VarDecl:
                Type: int @ 1:1
            BinaryOp: + @ 1:9
                Constant: int, 100 @ 1:9
                Constant: int, 50 @ 1:15
    GlobalDecl:
        Decl: ID(name=b)
            VarDecl:
                Type: int @ 2:1
            BinaryOp: * @ 2:10
                BinaryOp: + @ 2:10
                    Constant: int, 13 @ 2:10
                    ID: a @ 2:15
                Constant: int, 10 @ 2:20
